src/main/java/biz/k11i/xgboost/tree/PreorderRegTree.java

```diff
@@ -7,7 +7,6 @@
 import biz.k11i.xgboost.util.FVec;
 
 /**
- * author: adamj, niloyg
```

Comment:
why remove author line?

Reply:
I didn't write that, I'm just running `git apply`. Not sure why they wanted to remove it